src/store/playerActions.js

```diff
@@ -148,21 +148,22 @@ export const authenticateUser = () => (dispatch) => {
     }})
     .then(res => {            
         if (res.status === 200) {
+            connectSocket();
             dispatch(authSuccess(res.data.cookie));
         } else if (res.status === 401) {
-            dispatch(authFailure());
+            dispatch(authFailure("Unauthorised"));
             dispatch(resetState());
             dispatch(resetPlayerState());
         } else {
             const error = new Error(res.error);
-            dispatch(authFailure());
+            dispatch(authFailure(""));
```

Comment:
What does this do? Is it worth dispatching this with an empty string?

Reply:
No, but again it's left for whoever is doing the proper error handling (we haven't yet figure out how to deal with the situation in this else case)